src/RestClient/endpoints/V3/Themes/ThemeActions.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosInstance } from 'axios';
+
+import { themesPath } from './Themes';
+import type { ThemeAction } from './types';
+
+class ThemeActions {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Downloads a stores Theme.
+   *
+   * @param uuid A valid store Theme UUID
+   * @returns Promise resolving to a single theme
```

Comment:
This isn't accurate; the documentation actually describes that the response from this API method is a JSON Body with a single property: `job_id` (https://developer.bigcommerce.com/api-reference/24d4f462e909e-download-a-theme), which is "The job identifier for theme upload or download."